Add Deepfake Image Analyzer

vidurAgg22 opened the pull request:
Fixed issue assigned to me: #688
https://github.com/Niketkumardheeryan/ML-CaPsule/issues/688

work done:
- add a readme.md file
-add CNN deep fake image classifier
-add KNN deep fake image classifier
-add random forest deep fake image classifier
-add SVM deep fake image classifier

This is my first contribution so please review, add level tags, and let me know if something needs to be modified.

vidurAgg22 commented:
@invigorzz313 
Hi Bhavya, can you please have a look and let me know about modifications and assign level tags according to my work?

thankyou

Niketkumardheeryan commented:
Hi @vidurAgg22 ,Good works,and Good readme.md file  .Just you have to add .ipynb file your code with outputs that's it ...

vidurAgg22 commented:
@Niketkumardheeryan 
Hello Sir,

I have fixed the issue and added the .ipynb files. Please check and let me know if any more modifications are needed. Also, please add a level tag to it.

Thank you so much for your time.
